example-facets/films/templates/films/films_list.html

```diff
@@ -0,0 +1,106 @@
+{% extends 'base.html' %}
+{% load search_tags %}
+
+{% block title %}Films{% endblock %}
+
+{% block content %}
```

Comment:
Needs much cleanup. Trying to get a page to render so I can clean this bit up. (I feel like I need to see a rendering)